src/styles/components/_card.scss

```diff
@@ -0,0 +1,84 @@
+$card-border: 1px solid #e5e5e5;
```

Comment:
i think would be better move it into variables

Reply:
agree with @Nordexz, move it to `_variables`

Reply:
Fixed ✔